src/index.tsx

```diff
@@ -243,6 +276,77 @@ function getScrollbars(
   };
 }
 
+function handlerRectangles(
+  elementX1: number,
+  elementX2: number,
+  elementY1: number,
+  elementY2: number,
+  sceneState: SceneState
+) {
+  const margin = 4;
+  const minimumSize = 40;
+  const handlers: { [handler: string]: number[] } = {};
+
+  if (elementX2 - elementX1 > minimumSize) {
+    handlers["n"] = [
+      elementX1 + (elementX2 - elementX1) / 2 + sceneState.scrollX - 4,
+      elementY1 - margin + sceneState.scrollY - 8,
+      8,
+      8
+    ];
+
+    handlers["s"] = [
+      elementX1 + (elementX2 - elementX1) / 2 + sceneState.scrollX - 4,
+      elementY2 - margin + sceneState.scrollY + 8,
+      8,
+      8
+    ];
+  }
+
+  if (elementY2 - elementY1 > minimumSize) {
```

Comment:
What happens if you are currently dragging this one and it becomes too small, does it disappear and break the interaction?

Reply:
Actually, it's not a problem. Because you're dragging in the other direction, this one can never be too small while you're dragging as it's not changing size.

This is super cool that you can see it disappear while making it smaller! Really nice touch

Reply:
I think this never happens, because only the middle ones disappear, if you are resizing from north to south, you are hiding the west and east handle